changelogs/current.yaml

```diff
@@ -237,6 +237,11 @@ bug_fixes:
 - area: http2
   change: |
     Update nghttp2 to resolve CVE-2024-30255 https://github.com/envoyproxy/envoy/security/advisories/GHSA-j654-3ccm-vfmm.
+- area: outlier detection
+  change: |
+    Fixed
+    :ref:`successful_active_health_check_uneject_host<envoy_v3_api_field_config.cluster.v3.OutlierDetection.successful_active_health_check_uneject_host>`.
+    Before FAILED_ACTIVE_HC health flag was set, a failed health check as well could uneject the host.
```

Comment:
```suggestion
    Before, if ``FAILED_ACTIVE_HC`` health flag was set, a failed health check could uneject the host.
```

Reply:
im still a bit confused as to meaning wrt what is set/happens

is this correct?

>  Before, a failed health check could uneject the host if the `FAILED_ACTIVE_HC` had not been set

?

Reply:
Ok, I just added back the `health flag` phrase. I think it is useful.